ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1532,7 +1530,7 @@ - (instancetype)copyWithZone:(NSZone *)zone {
                                                                                     minimum:[_minimum copy]
                                                                                     maximum:[_maximum copy]
                                                                       maximumFractionDigits:[_maximumFractionDigits copy]];
-    answerFormat->_defaultNumericAnswer = [_defaultNumericAnswer copy];
+    answerFormat->defaultNumericAnswer = [_defaultNumericAnswer copy];
```

Comment:
won't this throw an error?

Reply:
Yes, don't know why the compiler didn't complain. My apologies.